anago

```diff
@@ -200,6 +200,17 @@ check_acls () {
 }
 
 ###############################################################################
+# Ensures that all registries that will be used during both mock and --nomock
+# runs allow write access so we don't fall over later
+# @param registries - A space separated list of registries
+#
+ensure_registry_acls () {
+  local registries=($1)
+
```

Comment:
@kubernetes/test-infra-maintainers You have all the tricks.  What's a good, fast way to ensure write access to registries?  docker push doesn't have a dryrun and the interwebs aren't giving any great solutions.